hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/sender/http/DefaultHttpMetadataAppender.java

```diff
@@ -8,8 +8,7 @@
 
     @Override
     public Request append(Request target, Message message) {
-
-        message.getExternalMetadata().forEach(target::header);
+        message.getExternalMetadata().forEach((header, value) -> target.getHeaders().put(header, value));
```

Comment:
what's the difference and purpose? is it `add` -> `set/override`?

Reply:
You're right, there is no difference. I had a problem with previous implementation (order for some reason mattered), now it's ok. I've reverted this change.